Resources/views/email/submissions.email.twig

```diff
@@ -0,0 +1,111 @@
+            }
+
+            .email-intro {
+                font-size: 14px !important;
+                line-height: 21px !important;
+            }
+        }
+
+        @media screen and (min-width: 321px) and (max-width: 375px) {
+            .email-title {
+                font-size: 22px !important;
+                line-height: 36px !important;
+            }
+
+        }
+
+        .client-link a {
+            color: #3a383a !important;
+        }
+
+        td {
+            -ms-text-size-adjust: none;
+            -webkit -text-size-adjust: none;
+        }
+
+        table {
+            margin: 0 auto !important;
+        }
+
+    </style>
+    <!--[if gte mso 12]>
+    <style>
+        td {
+            mso-line-height-rule: exactly;
+        }
+    </style>
+    <![endif]-->
+</head>
+<body style="margin: 0; padding: 0;">
+<table border="0" cellpadding="0" cellspacing="0" width="100%">
+    <tr>
+        <td style="padding: 0 10px;">
+            <!--[if (mso)|(IE)]>
+            <table align="center" border="0" cellpadding="0" cellspacing="0" width="600">
+                <tr>
+                    <td>
+            <![endif]-->
+            <table align="center" border="0" cellpadding="0" cellspacing="0" class="width-full"
+                   style="max-width: 600px;" width="100%">
+                <tr>
+                    <td class="email-title"
+                        style="color: #3a383a; font-family: Georgia, serif; font-size: 42px; font-weight: bold; line-height: 44px; padding-top: 20px; text-transform: uppercase;">
+                        Submissions
+                    </td>
+                </tr>
+                <tr>
+                    <td class="email-intro"
+                        style="color: #3a383a; font-family: Georgia, serif; font-size: 18px; line-height: 27px; padding-top: 5px;">
+                        <table border="1">
+                            <tr>
+                                <th>Label</th>
+                                <th>Submission Date</th>
+                                <th>Expiry Date</th>
+                                {% for key, value in submissions.0.data %}
+                                    <th> {{ key }}</th>
+                                {% endfor %}
+                            </tr>
+                            {% for submission in submissions %}
+                                <tr>
+                                    <td>{{ submission.label }}</td>
+                                    <td>{{ submission.created|date('d/m/Y') }}</td>
+                                    <td>{{ submission.expireDate|date('d/m/Y') }}</td>
+                                    {% for key, value in submission.data %}
+                                        {% if value  is not iterable %}
+                                            <td>{{ value }}</td>
+                                        {% endif %}
+                                    {% endfor %}
```

Comment:
What value would that be?

Reply:
The title of the column, for example "firstname" or "email", but as discussed will be removed.